index.bs

```diff
@@ -1015,15 +1025,19 @@ enum MLOperandDataType {
 };
 
 dictionary MLOperandDescriptor {
-  // The operand type.
   required MLOperandDataType dataType;
-
-  // The dimensions field is empty for scalar operands,
-  // and non-empty for tensor operands.
   sequence<[EnforceRange] unsigned long> dimensions = [];
 };
 </script>
 
+<dl dfn-type=dict-member dfn-for=MLOperandDescriptor>
+    : <dfn>dataType</dfn>
+    :: The operand type.
```

Comment:
Nit:
```suggestion
    :: The operand data type.
```

Reply:
Done in 1f1b3c13762ec4366ebb9e233fec09f6ae432ddf